src/main/resources/mainnet.conf

```diff
@@ -57,9 +57,9 @@ ergo {
 
     # Optional and individual checkpoint.
     # Before the height given (including it) validation of scripts is missed.
-    # This improving perfomance and memory usage during initial bootstrapping.
```

Comment:
The start of the checkpoint doc is very confusing, meaning of checkpoint should be explained here

Reply:
Improved this comment a bit, please check

Reply:
This hit the spot, thanks :+1: